qa/TL0_multigpu/test_jax.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash -e
+# used pip packages
+pip_packages='${python_test_runner_package} numpy jax'
+
+target_dir=./dali/test/python
+
+# test_body definition is in separate file so it can be used without setup
+source test_body.sh
+
+test_body() {
+  test_jax
+}
+
+# run this only on x86_64, not arm
+if [ $(uname -m) != "x86_64" ]
```

Comment:
Is JAX not supported on aarch64 (I think TF is).

Reply:
Affirmative, there is no official JAX support or package distribution for ARM. https://github.com/google/jax/issues/5501